### PR TITLE
fix:  #419 prevent survey in the first loading screen

### LINKED ISSUE #419: NPS survey "recommend to a friend" popup can appear in loading screen
The dialog is showing in loading screen. When the loading takes too much time it can be observed.

It should only appear when:
- Rendering is activated (loading screen not shown)
- Some time has passed in the world exploration state

On top of fixing that, let's configure Delighted so that the survey only appears from the 2nd play session onwards (or from the 2nd day, both are OK).

### PR DESCRIPTION
**Pull request by 2fd**
Fix #419

# What? 
Prevent display delighted's survey

### CODE REVIEW

File: kernel/packages/unity-interface/dcl.ts [hunk 1]
```diff
@@ -194,13 +195,14 @@ export function setLoadingScreenVisible(shouldShow: boolean) {
   document.getElementById('load-messages-wrapper')!.style.display = shouldShow ? 'block' : 'none'
   document.getElementById('progress-bar')!.style.display = shouldShow ? 'block' : 'none'
   if (!shouldShow) {
+    isTheFirstLoading = false
```
Comment (BrianAmadori) [inline kernel/packages/unity-interface/dcl.ts:204]: When the user is in tutorial flow, this wouldn't show the survey popup as soon as the 2nd teleport is triggered? Chat and other features will not be available yet. Are we sure this is the intended behaviour?

Comment (moliva) [reply kernel/packages/unity-interface/dcl.ts:204]: I agree with Brian, maybe we need another criterion to show this survey or a combination of them, e.g. finished tutorial + certain elapsed in world or teleport

Comment (2fd) [reply kernel/packages/unity-interface/dcl.ts:204]: I agree with you guys, this isn't the optimal solution but was the best effort/benefit. I have other priorities, but you are free to implement these changes

**Review by BrianAmadori — APPROVED** [review]

File: kernel/packages/unity-interface/dcl.ts (hunk 1 above)
Comment (BrianAmadori) [reply kernel/packages/unity-interface/dcl.ts:204]: Ok, just approving it. We may improve this in the future.

**Review by moliva — APPROVED** [review]

**Pull request merged**